lib/faerie/src/backend.rs

```diff
@@ -3,20 +3,22 @@
 use container;
 use cretonne_codegen::binemit::{Addend, CodeOffset, Reloc, RelocSink, NullTrapSink};
 use cretonne_codegen::isa::TargetIsa;
-use cretonne_codegen::result::CtonError;
 use cretonne_codegen::{self, binemit, ir};
-use cretonne_module::{Backend, DataContext, Linkage, ModuleNamespace, Init, DataDescription};
-use faerie;
+use cretonne_module::{Backend, DataContext, Linkage, ModuleNamespace, Init, DataDescription,
+                      ModuleError};
 use failure::Error;
+use faerie;
 use std::fs::File;
 use target;
+use traps::{FaerieTrapManifest, FaerieTrapSink};
 
 /// A builder for `FaerieBackend`.
 pub struct FaerieBuilder {
     isa: Box<TargetIsa>,
     name: String,
     format: container::Format,
     faerie_target: faerie::Target,
+    collect_traps: bool,
```

Comment:
Would you mind introducing a custom enum here, rather than using bool, so make end user code easier to read?